app/forms/questionnaire_form.py

```diff
@@ -336,7 +343,12 @@ def get_answer_fields(question, data, error_messages, answer_store, metadata, lo
                 ).get_field()
 
         answer_fields[answer["id"]] = get_field_handler(
-            answer, error_messages, answer_store, metadata, location
+            answer,
+            error_messages,
+            answer_store,
+            metadata,
+            location,
+            question_title=question.get("title"),
```

Comment:
For consistency `question_title` should also be passed to the detail answer `get_field_handler` just before this.